ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/fragmentPage/v1/fragmentPage/body.jsp

```diff
@@ -0,0 +1,21 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
no JSP files in core components, everything has to be in sightly

Reply:
Yes, this will be removed as this was for static implementation.